app/test_main.py

```diff
@@ -1,3 +1,21 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_two_parameters_equal_to_zero() -> None:
+    assert (get_human_age(0, 0) == [0, 0])
+
+
+def test_one_parameter_equal_to_zero() -> None:
+    assert (get_human_age(0, 21) == [0, 1])
+
+
+def test_float_value_of_parameters() -> None:
+    assert (get_human_age(23.2, 25) == [1, 2])
+
+
+def test_very_big_value_of_parameters() -> None:
+    assert (get_human_age(110324, 13455) == [27577, 2688])
```

Comment:
The test `test_very_big_value_of_parameters` checks for large input values. Ensure that the function `get_human_age` is designed to handle such large numbers without performance issues.